packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -42,33 +44,41 @@ export default function AnnualHighlightCards( {
 
 	return (
 		<div className={ classNames( 'highlight-cards', className ?? null ) }>
-			{ header }
+			<div className="highlight-year-navigation">
+				{ header }
+				{ navigation && navigation }
+			</div>
 
 			<div className="highlight-cards-list">
 				<HighlightCard
 					heading={ translate( 'Posts' ) }
 					icon={ <Icon icon={ postContent } /> }
 					count={ counts?.posts ?? null }
+					showValueTooltip
```

Comment:
This limits the tooltips to the highlight component on the `Insights` page and prevents them from showing up on the main `Traffic` page.